luizfm/redux-challenge-004
# CHA-0002 login integration

## More details
- Added redux integration and removed a lot of unnecessary things like webpack and babel.

## How has this been tested?
- With a cloned repository, do a ```yarn
- Wait until the packages have been installed and run a ```yarn start``` on your terminal.
- After that, do a ```yarn json-server db.json -m ./node_modules/json-server-auth -r routes.json -p 3333```
- Try to login with any credentials. An error should appear
- Try to login with ```email: jungle.test@jungledevs.com; password: sup3rpassw0rd```. A console log "ok true" should appear on your console.

## Screenshots
![Screen Shot 2021-04-30 at 10 45 28 AM](https://user-images.githubusercontent.com/45155140/116703920-30e3db00-a9a1-11eb-9f77-67667478fd2d.png)
![Screen Shot 2021-04-30 at 10 46 22 AM](https://user-images.githubusercontent.com/45155140/116704019-4fe26d00-a9a1-11eb-9909-1cb7afd164aa.png)
